packages/fast-components-react-base/src/breadcrumb/breadcrumb.tsx

```diff
@@ -0,0 +1,111 @@
+import { get } from "lodash-es";
+
+class Breadcrumb extends Foundation<
+    BreadcrumbHandledProps,
+    BreadcrumbUnhandledProps,
+    {}
+> {
+    public static displayName: string = "Breadcrumb";
+
+    protected handledProps: HandledProps<BreadcrumbHandledProps> = {
+        children: void 0,
+        label: void 0,
+        separator: void 0,
+        managedClasses: void 0,
+    };
+
+    /**
+     * Renders the component
+     */
+    public render(): React.ReactElement<HTMLElement> {
+        return (
+            <nav
+                {...this.unhandledProps()}
+                aria-label={this.props.label || null}
+                className={this.generateClassNames()}
+            >
+                <ol className={this.generateOlClassNames()}>{this.renderChildren()}</ol>
+            </nav>
+        );
+    }
+
+    /**
+     * Create class names
+     */
+    protected generateClassNames(): string {
+        return super.generateClassNames(get(this.props.managedClasses, "breadcrumb"));
+    }
+
+    /**
+     * Create class Ol names
+     */
+    protected generateOlClassNames(): string {
+        return get(this.props.managedClasses, "breadcrumb_itemsContainer");
+    }
+
+    /**
+     * Render all child elements
+     */
+    private renderChildren(): React.ReactFragment[] {
+        return React.Children.map(this.props.children, this.renderChild);
+    }
+
+    private isClonableElement(node: React.ReactNode): node is React.ReactElement<any> {
+        return React.isValidElement(node);
+    }
+
+    private generateCurrentClassName(): string {
+        const className: string = this.props.managedClasses.breadcrumb_item__current;
+        return className !== undefined ? className : "";
+    }
+    /**
+     * Render a single child
+     */
+    private renderChild = (
+        child: React.ReactNode,
+        index: number
+    ): React.ReactFragment => {
+        const childCount: number = React.Children.count(this.props.children);
+        let augmentedChild: React.ReactNode = child;
+        let notLastItem: boolean = true;
+        if (this.isClonableElement(child)) {
+            const props: any = {
+                className:
+                    child.props && typeof child.props.className === "string"
+                        ? `${child.props.className} ${
+                              this.props.managedClasses.breadcrumb_item
+                          }`
+                        : this.props.managedClasses.breadcrumb_item,
+            };
+
+            if (childCount - 1 === index) {
+                props.className = `${props.className} ${this.generateCurrentClassName()}`;
+                props["aria-current"] = "page";
+                notLastItem = false;
+            }
+
+            augmentedChild = React.cloneElement(child, props);
+        }
+        return (
+            <li>
+                {augmentedChild}
+                {typeof this.props.separator === "function" && notLastItem

```

Comment:
You should use `get(this.props, "managedClasses.breadcrumb_separator"` so that it doesn't throw if managedClasses are not provided

Reply:
already fixed.